lib/Signaling/Listener.php

```diff
@@ -324,6 +327,88 @@ public static function notifyParticipantsAfterGuestClean(RoomEvent $event): void
 		$notifier->participantsModified($event->getRoom(), $sessionIds);
 	}
 
+	public static function notifyParticipantsAfterSetBreakoutRoomStatus(RoomEvent $event): void {
+		if (self::isUsingInternalSignaling()) {
+			return;
+		}
+
+		$room = $event->getRoom();
+		if ($room->getBreakoutRoomStatus() === BreakoutRoom::STATUS_STARTED) {
+			self::notifyParticipantsAfterBreakoutRoomStarted($room);
+		} else {
+			self::notifyParticipantsAfterBreakoutRoomStopped($room);
+		}
+	}
+
+	private static function notifyParticipantsAfterBreakoutRoomStarted(Room $room): void {
+		$manager = Server::get(Manager::class);
+		$breakoutRooms = $manager->getMultipleRoomsByObject(BreakoutRoom::PARENT_OBJECT_TYPE, $room->getToken());
+
+		$switchToData = [];
+
+		$participantService = Server::get(ParticipantService::class);
+		$parentRoomParticipants = $participantService->getSessionsAndParticipantsForRoom($room);
+
+		$notifier = Server::get(BackendNotifier::class);
+
+		foreach ($breakoutRooms as $breakoutRoom) {
+			$sessionIds = [];
+
+			$breakoutRoomParticipants = $participantService->getParticipantsForRoom($breakoutRoom);
```

Comment:
https://github.com/nextcloud/spreed/pull/8660 brings in a `getSessionsAndParticipantsForRooms` so we can get them all with one query instead of a query per room
Maybe something for a follow up